packages/collaboration/style/sidepanel.css

```diff
@@ -142,3 +142,37 @@
   box-shadow: 0 2px 2px -2px rgb(0 0 0 / 24%);
 
 }
+
+/************************************************************
+                User Info Details
+*************************************************************/
+.jp-UserInfo-Field {
+  display: flex;
+  justify-content: space-between;
+}
+
+.jp-UserInfo-Field > * {
```

Comment:
Can the `*` be replaced with a more precise selector? `*` has negative performance implications as every DOM element will be evaluated against the path.

Reply:
I updated it with the expected tags.